src/main/java/earl/Earl.java

```diff
@@ -0,0 +1,94 @@
+package earl;
+
+import earl.exceptions.EarlException;
+import earl.logic.HandlerFactory;
+import earl.util.Parser;
+import earl.util.Storage;
+import earl.util.TaskList;
+import earl.util.Ui;
+
+/**
+ * The main class of the Earl application.
+ */
+public class Earl {
+
+    private final Storage storage;
+    private final TaskList tasks;
+    private final Ui ui;
+
+    /**
+     * Constructor of the Earl class.
+     *
+     * @param filePath a path to the text file storing past data
+     */
+    public Earl(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        TaskList temp;
```

Comment:
Should the variable name be changed to a more descriptive name? The use of 'temp' is also seen in a few other places.

Reply:
Thank you for the comments! I'll get to fixing it.